packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -75,6 +75,28 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     this.selection.setItems(this.state.items);
   }
 
+  public componentWillReceiveProps(newProps: P) {
+    let newItems = newProps.selectedItems;
```

Comment:
Ah! Yes. 

Should selectedItems override the default items? I think it should.